### PR TITLE
Feanil/abbey common

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

File: playbooks/roles/datadog/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - common
```
Comment (jarv) [inline playbooks/roles/datadog/meta/main.yml:3]: datadog is running its own supervisor right?

File: util/vpc-tools/abbey.py [hunk 2]
```diff
@@ -133,10 +134,10 @@ def parse_args():
                         help="Application for subnet, defaults to admin",
                         default="admin")
     parser.add_argument('--configuration-version', required=False,
-                        help="configuration repo version",
+                        help="configuration repo branch(no hashes)",
```
Comment (jarv) [inline util/vpc-tools/abbey.py:137]: why not?

File: util/vpc-tools/abbey.py [hunk 3]
```diff
@@ -438,7 +441,10 @@ def poll_sqs_ansible():
                         to_disp['msg']['TS'] / 60,
                         to_disp['msg']['TS'] % 60,
                         to_disp['msg']['PREFIX'])
-                    return (to_disp['msg']['TS'], task_report)
+
+                    completed += 1
+                    if completed >= 3:
```
Comment (jarv) [inline util/vpc-tools/abbey.py]: what's this logic for? can you add a comment?

File: playbooks/roles/datadog/meta/main.yml (hunk 1 above)
Comment (feanil) [inline playbooks/roles/datadog/meta/main.yml:3]: Yea, that is all managed by the datadog apt package.

File: util/vpc-tools/abbey.py (hunk 2 above)
Comment (feanil) [inline util/vpc-tools/abbey.py:137]: The '-b' option of `git clone` only takes valid branch names.  It doesn't accept hashes or other gitrefs.

**Comment by jarv** [comment]
:+1:

**Pull request merged**